web/src/pages/SettingsPage.scss

```diff
@@ -43,3 +43,10 @@
     flex-wrap: wrap;
   }
 }
+
+button.parent-dir {
```

Comment:
Currently this also affects the up button of the file browser, causing it to move to the top-left of the page. I think it might be nice to move this to a modal-specific stylesheet to prevent it from any other side effects.